packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1043,13 +1043,12 @@ curl -v -X POST \
 
 #### Add Smart Card Identity Provider
 
-
 Adds a new Smart Card `X509` type IdP to your organization
 
-##### Request Example
+##### Request example
 
-> **Notes:** You must first add the IdP's server certificate to the IdP key store before you can add a Smart Card `X509` IdP with a `kid` credential reference. You need to upload the whole trust chain as a single key using the [Key Store API](#add-x-509-certificate-public-key).
-Depending on the information stored in the smart card select the proper [template](/docs/reference/okta-expression-language/#idp-user-profile) `idpuser.subjectAltNameEmail` or `idpuser.subjectAltNameUpn`.
+> **Notes:** You must first add the IdP's server certificate to the IdP key store before you can add a Smart Card `X509` IdP with a `kid` credential reference. You need to upload the whole trust chain as a single key using the [Key Store API](#add-x-509-certificate-public-key).<br>
+Depending on the information stored in the Smart Card, select the proper [template](/docs/reference/okta-expression-language/#idp-user-profile) `idpuser.subjectAltNameEmail` or `idpuser.subjectAltNameUpn`.
```

Comment:
Not sure this should be capitalized, looks like it's referring to the physical smart card itself.